v-next/hardhat-ignition-ui/.eslintrc.cjs

```diff
@@ -1,14 +1,11 @@
-module.exports = {
-  env: { browser: true, es2020: true },
-  extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:react-hooks/recommended',
-  ],
-  parser: '@typescript-eslint/parser',
-  parserOptions: { ecmaVersion: 'latest', sourceType: 'module' },
-  plugins: ['react-refresh'],
-  rules: {
-    'react-refresh/only-export-components': 'warn',
-  },
-}
+const { createConfig } = require("../../config-v-next/eslint.cjs");
+
+const config = createConfig(__filename);
+
+delete config.rules["@typescript-eslint/naming-convention"];
```

Comment:
There is an eslint exception if we don't remove the rule.
I suspect this might be an odd interaction between our eslint rules and the particularities of the ignition ui codebase.
The eslint rules need revisited.